client/state/comments/actions.js

```diff
@@ -56,21 +56,20 @@ export const requestCommentsList = query => ( {
 	query,
 } );
 
-/***
- * Creates a remove comment action for a siteId, postId, commentId
+/**
+ * Creates an action that permanently deletes a comment
```

Comment:
This isn't blocking, I'm just curious on behavior here, and to update the docs if it's different from what we expect.